modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="getting-started-with-red-hat-openshift-gitops_{context}"]
+= Getting Started with Red Hat OpenShift GitOps
+
+[id="installing-red-hat-openshift-gitops_{context}"]
+== Installing Red Hat OpenShift GitOps
+
+.Prerequisite
```

Comment:
Please use a bulleted list for Prerequisites, see here: https://docs.openshift.com/container-platform/4.6/pipelines/creating-applications-with-cicd-pipelines.html#prerequisites

Reply:
Add the prereq with links into the assembly. that way, you're not adding any links into the module.